cmd/aws-k8s-agent/main.go

```diff
@@ -16,29 +16,27 @@ package main
 import (
 	"os"
 
-	log "github.com/cihub/seelog"
-
 	"github.com/aws/amazon-vpc-cni-k8s/pkg/eniconfig"
 	"github.com/aws/amazon-vpc-cni-k8s/pkg/ipamd"
 	"github.com/aws/amazon-vpc-cni-k8s/pkg/k8sapi"
 	"github.com/aws/amazon-vpc-cni-k8s/pkg/utils/logger"
 )
 
-const (
-	defaultLogFilePath = "/host/var/log/aws-routed-eni/ipamd.log"
-)
+const binaryName = "L-IPamD"
```

Comment:
I think `ipamd` is a fine abbreviation of "IP Address Management Daemon". The Linux part is not needed, and neither the special casing.

Reply:
Heh, TIL that the "L-" stands for "Linux"...

Agreed, though, that "ipamd" is a better and more Linux-appropriate daemon name.

If you make this change in a future PR, however, please remember to update https://github.com/awslabs/amazon-eks-ami/blob/master/log-collector-script/linux/eks-log-collector.sh

Reply:
> I think `ipamd` is a fine abbreviation of "IP Address Management Daemon". The Linux part is not needed, and neither the special casing.

I wanted it to be consistent with the [log message](https://github.com/aws/amazon-vpc-cni-k8s/blob/master/cmd/aws-k8s-agent/main.go#L43). I can change it to "ipamd".